Library/Homebrew/cask/cask.rb

```diff
@@ -169,6 +169,8 @@ def eql?(other)
     def to_h
       {
         "token"          => token,
+        "full_token"     => "#{tap.name}/#{token}",
+        "tap"            => tap.name,
```

Comment:
```suggestion
        "tap"            => tap&.name,
```

`tap` can be `nil` so let's ensure there are no errors thrown in those cases

Reply:
Perfect! I'll make the change!

Reply:
Done!